src/components/LayoutTree.tsx

```diff
@@ -591,6 +591,11 @@ const LayoutTree = () => {
   // get the scroll position before the render so it can be preserved
   const scrollY = window.scrollY
 
+  // When a thought is hovered over a sorted context, determine the position of arrow.
+  useEffect(() => {
+    calculateThoughtPosition(treeThoughtsPositioned)
+  }, [calculateThoughtPosition, treeThoughtsPositioned])
```

Comment:
I first put it in `LayoutTree` but the arrow was not staying fixed relative to the viewport during scroll. So I put it in `Content`, I really don't like this way as well since I have to pass props to `LayoutTree` which really does not need to be accepting it.

Reply:
It's going to have to be switched to `position: absolute` anyway to make it work on mobile Safari.